### PR TITLE
Update scripts for GCE

### PR DESCRIPTION
**Pull request by AlCutter**
A simple script to update the log/mirror jobs.

### CODE REVIEW

File: cloud/google/update_log.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Header "Updating log instances..."
+# TODO(alcutter): Wait for each task (or zone?) to be healthy before
+# proceeding.
+for i in `seq 0 $((${LOG_NUM_REPLICAS} - 1))`; do
+  echo "Updating ${LOG_MACHINES[${i}]}"
+  gcloud compute ssh ${LOG_MACHINES[${i}]} \
+      --zone ${LOG_ZONES[${i}]} \
+      --command \
+          'sudo docker pull gcr.io/'${PROJECT}'/super_duper:test &&
```
Comment (phad) [inline cloud/google/update_log.sh]: What is gcr.io ?

File: cloud/google/update_mirror.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Header "Updating mirror instances..."
+# TODO(alcutter): Wait for each task (or zone?) to be healthy before
+# proceeding.
+for i in `seq 0 $((${MIRROR_NUM_REPLICAS} - 1))`; do
+  echo "Updating ${MIRROR_MACHINES[${i}]}"
+  gcloud compute ssh ${MIRROR_MACHINES[${i}]} \
+      --zone ${MIRROR_ZONES[${i}]} \
+      --command \
+          'sudo docker pull gcr.io/'${PROJECT}'/super_mirror:test &&
```
Comment (phad) [inline cloud/google/update_mirror.sh]: Was s/duper/mirror/g the only change between these two files?
I think you might find some common stuff to extract in that case :)

File: cloud/google/update_log.sh (hunk 1 above)
Comment (AlCutter) [inline cloud/google/update_log.sh]: It's Google's new container registry, docs at... https://gcr.io

File: cloud/google/update_mirror.sh (hunk 2 above)
Comment (AlCutter) [inline cloud/google/update_mirror.sh]: `/MIRROR_/LOG_/` too.
But the reasons I didn't factor it out are:
 - passing arrays is painful in bash, though I could just copy the contents of the `LOG_*` and `MIRROR_*` to e.g. `NODE_*` before calling, but
 -  when the `TODO`s are `TODONE`, there could be different checks necessary depending on you're updating (especially if that could also be an `etcd` instance.), again I guess you could define a function and then pass in the command to call it which gets `eval`'d  or something, but you have to question whether it's worth it to dedupe this pretty small loop...

**Pull request merged**